integration-tests/api/__tests__/store/cart/cart.js

```diff
@@ -2263,6 +2285,10 @@ describe("/store/carts", () => {
   describe("get-cart with session customer", () => {
     beforeEach(async () => {
       await cartSeeder(dbConnection)
+      await simpleSalesChannelFactory(dbConnection, {
```

Comment:
suggestion(non blocking): I ve seen a lot of that and it seams that we can just call the storeFactory. That way it is also attached to the store like expected. wdyt?

Reply:
We don't have a store factory

Reply:
Is this not in the plugins integration tests suite?

Reply:
ahah you are right, it is in the plugin. 👍